install/installer/pkg/config/v1/envvars_test.go

```diff
@@ -30,6 +30,8 @@ type envvarTestData struct {
 }
 
 func TestBuildFromEnvvars(t *testing.T) {
+	t.Skip("--remove--")
```

Comment:
Should this be included in the PR? IF we want to remove it, does it make sense to remove directly in this PR?

Reply:
fixed